src/scss/layout.scss

```
@@ -60,6 +60,14 @@
   margin-bottom: $default-margin * 2;
 }
 
+.index-article {
+  background: white;
+  border-radius: 5px;
+  padding: 2em 2em 2em 2em;
+  border-radius: 5px;
+  margin: -1rem 0 0 0;
```

You can use shorthand

```suggestion
  margin: -1rem 0 0;
```